app/assets/config/manifest.js

```diff
@@ -1,3 +1,2 @@
 //= link_tree ../images
-//= link_directory ../javascripts .js
```

Comment:
This line had to be removed before there is no `app/assets/javascripts`, JavaScripts are managed by yarn.